src/js-ipfs-api/index.test.js

```diff
@@ -1,14 +1,14 @@
 /* global it, expect, jest */
 const tryApi = require('./index.js')
 const { URL } = require('url')
-const IpfsApi = require('ipfs-http-client')
+const ipfsClient = require('ipfs-http-client')
```

Comment:
We may add other clients (when we have something like a realtime RPC API over websockets).
Let's be specific that this is HTTP one:
```suggestion
const httpClient = require('ipfs-http-client')
```

Reply:
Done :D